docs/pages/versions/unversioned/sdk/background-task.mdx

```diff
@@ -0,0 +1,208 @@
+  }, []);
+
+  const checkStatusAsync = async () => {
+    const status = await BackgroundTask.getStatusAsync();
+    setStatus(status);
+  };
+
+  const toggle = async () => {
+    if (isRegistered) {
+      await registerBackgroundTaskAsync();
+    } else {
+      await unregisterBackgroundTaskAsync();
+    }
+    setIsRegistered(!isRegistered);
+  };
+
+  return (
+    <View style={styles.screen}>
+      <View style={styles.textContainer}>
+        <Text>
+          Background Task Service Availability:{' '}
+          <Text style={styles.boldText}>
+            {status ? BackgroundTask.BackgroundTaskStatus[status] : null}
+          </Text>
+        </Text>
+      </View>
+      <Button
+        disabled={status === BackgroundTask.BackgroundTaskStatus.Restricted}
+        title={isRegistered ? 'Cancel Background Task' : 'Schedule Background Task'}
+        onPress={toggle}
+      />
+      <Button title="Check Background Task Status" onPress={checkStatusAsync} />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  screen: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  textContainer: {
+    margin: 10,
+  },
+  boldText: {
+    fontWeight: 'bold',
+  },
+});
+```
+
+## Testing background tasks
+
+Background tasks can be difficult to test because they can be triggered inconsistently by the system. Fortunately, Expo Background Task contains some tools that help make testing background tasks easier.
+
+### Android&ensp;<PlatformTags platforms={['android']} />
+
+On Android, the easiest way to test background tasks is to register a task. When the app is in `debug` mode, a background task should be started as soon as possible, while in release mode, it will wait until the minimum interval has passed before running the task.
+
+> In release mode, the app must be in the background for the background task to run, while in debug mode, the task will run even if the app is in the foreground.
+
+### iOS&ensp;<PlatformTags platforms={['ios']} />
+
+On iOS, testing background tasks is done by [simulating](https://developer.apple.com/documentation/backgroundtasks/starting-and-terminating-tasks-during-development#Launch-a-Task) the launch of the task. This can be done by calling the `triggerTaskWorkerForTestingAsync` method in JavaScript.
+
+Expo Background Task has a convenient helper on iOS that triggers executing the background task worker. This will only work in `debug` mode:
+
+```ts
+import * as BackgroundTask from 'expo-background-task';
+
+export const triggerBackgroundTask = async () => {
+  await BackgroundTask.triggerTaskWorkerForTestingAsync();
+};
+```
+
+## Multiple background tasks
+
+Since the Background Tasks API on iOS and the WorkManager API on Android limit the number of tasks that can be scheduled for a single app, Expo Background Task uses a single worker on both platforms. While you can define multiple JavaScript background tasks, they will all run through this single worker.
```

Comment:
```suggestion
Since the WorkManager API on Android and the Background Tasks API on iOS limit the number of tasks that can be scheduled for a single app, Expo Background Task uses a single worker on both platforms. While you can define multiple JavaScript background tasks, they will all run through this single worker.
```